### PR TITLE
feat-utility

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (84)

### CODE REVIEW

File: stylesheets/components/utility.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import url(/stylesheets/main.css);
+
+.center-text{
+     position: relative;
```
Comment (rahulcs754) [inline stylesheets/components/utility.css]: you don't need this position

**Review by rahulcs754 — COMMENTED** [review]
I looked at your code. you can also created text-right ,  text-left  that will help you in future

File: stylesheets/components/utility.css (hunk 1 above)
Comment (iamsaurav28) [reply stylesheets/components/utility.css]: ok done as u suggested

**Review by rahulcs754 — APPROVED** [review]
Please remove the space here I can see.

**Pull request merged**